aldryn_config.py

```diff
@@ -365,16 +365,16 @@ def sentry_settings(self, settings, env):
         sentry_dsn = env('SENTRY_DSN')
 
         if sentry_dsn:
-            settings['INSTALLED_APPS'].append('raven.contrib.django')
-            settings['RAVEN_CONFIG'] = {
-                'dsn': sentry_dsn,
-                'release': env('GIT_COMMIT', 'develop'),
-                'environment': env('STAGE', 'local'),
-            }
-            settings['LOGGING']['handlers']['sentry'] = {
-                'level': 'ERROR',
-                'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
-            }
+            import sentry_sdk
+            from sentry_sdk.integrations.django import DjangoIntegration
```

Comment:
remove `from` include and just us `sentry_sdk.integrations.django import DjangoIntegration`below